src/components/common/names-capture.vue

```diff
@@ -315,35 +342,35 @@ export default class NamesCapture extends NameRequestMixin {
     // eg, when user continues to send for review
     this.$el.addEventListener('keydown', this.handleKeydown)
 
-    newReqModule.mutateNameChoicesToInitialState()
-    if (this.isAssumedName && !this.editMode) {
+    this.setNameChoicesToInitialState(null)
+    if (this.getIsAssumedName && !this.getEditMode) {
       this.$nextTick(() => { this.hide = true })
       return
     }
 
     this.$nextTick(() => {
-      if (this.editMode) {
+      if (this.getEditMode) {
```

Comment:
FWIW, this code allows execution to continue to lines 356-375, and after that it executes this arrow function. So quite likely this isn't working as intended.

This could initially be solved with an `await this.nextTick()`, or ultimately with better async design.

I'm going to fix this now because there's the same issue on line 347...